src/pages/student/Dashboard.js

```diff
@@ -525,7 +525,16 @@ function Dashboard({ student }) {
               </i>
             </span>
           </div>
-          <div className="dashboard-events-photo">
+
+          <div className="dashboard-events-photo-blob">
+            <img
+              className="dashboard-blobblue-img"
+              src={require('../assets/blobblue.png').default}
+              alt="blob blue"
+            />
+          </div>
```

Comment:
Pending what Allen has to say about keeping the stretched pngs vs finding the exact blobs, I have some suggestions if we do move forward with the stretched png. I think we can just use the img class (as opposed to have the div class and the image class) just so we trim down how many classes we have in the css. These lines could be replaced with the following suggestion, and I'll leave the corresponding comments in Dashboard.scss as well.

```suggestion

          <img
            className="dashboard-blobblue-img"
            src={require('../assets/blobblue.png').default}
            alt="blob blue"
          />
```